rest-api/data_gen/fake_participant_generator.py

```diff
@@ -489,13 +489,17 @@ def _submit_hpo_changes(self, participant_response, participant_id, consent_time
 
   def _submit_status_changes(self, participant_response, participant_id, last_request_time):
```

Comment:
The participant_response parameter can be removed I think.

Reply:
Done.